src/main/resources/com/mysql/cj/LocalizedErrorMessages.properties

```diff
@@ -444,7 +444,7 @@ ReplicationGroupManager.0=Unable to register replication host management bean wi
 ResultSet.Retrieved__1=Retrieved 
 ResultSet.Bad_format_for_BigDecimal=Bad format for BigDecimal ''{0}'' in column {1}.
 ResultSet.Bad_format_for_BigInteger=Bad format for BigInteger ''{0}'' in column {1}.
-ResultSet.Column_Index_out_of_range_low=Column Index out of range, {0} < 1.
+ResultSet.Column_Index_out_of_range_low=Column Index out of range, {0} < {1}.
```

Comment:
@sergiyvamz found an problem with an unparameterized part of another error code here